debian/rules

```diff
@@ -7,6 +7,7 @@ export DEB_LDFLAGS_MAINT_APPEND = -Wl,-O1 -Wl,--as-needed
 export DEB_LDFLAGS_MAINT_STRIP := -Wl,-Bsymbolic-functions
 
 GS_CONFIGURE_FLAGS = \
+	-Dfwupd=true \
```

Comment:
Please don’t touch the `-Dfwupd` settings here or in the “these are conditionally re-enabled later” section below, as those sections come from upstream Debian packaging, and any changes we make to them make the rebase a little harder.

You need to set `-Dfwupd-true` in the “EOS-specific configuration section” further down.

Reply:
Thanks for your review!  That makes sense, so I update the commit.  Please review again.  Thank you!